web/src/components/AppBar/UserMenu.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <!-- maybe...
+    :close-on-content-click="false"
+  -->
+  <v-menu
+    offset-y
+  >
+    <template #activator="{ on }">
+      <v-btn
+        icon
+        v-on="on"
+      >
+        <v-avatar
+          color="light-blue lighten-4"
+        >
+          <span
+            class="primary--text"
+          >
+            {{ userInitials }}
+          </span>
+        </v-avatar>
+      </v-btn>
+    </template>
+    <v-list
+      dense
+    >
+      <v-list-item>
+        <v-list-item-content>
+          Get Girder API Key
+        </v-list-item-content>
+        <v-list-item-action>
+          <v-icon>mdi-key</v-icon>
+        </v-list-item-action>
+      </v-list-item>
```

Comment:
Is this supposed to be hooked up to any click action currently?

Reply:
Eventually, but that's a WIP that I'll have to fix later.